tools/markdown-magic/src/utilities.cjs

```diff
@@ -95,13 +95,19 @@ function getPackageMetadata(packageJsonFilePath) {
  * Gets the appropriate special scope kind for the provided package name, if applicable.
  *
  * @param {string} packageName
- * @returns {"FRAMEWORK" | "EXPERIMENTAL" | "INTERNAL" | "PRIVATE" | "TOOLS" | undefined} A scope kind based on the package's scope (namespace).
- * Will be `undefined` if the package has no scope, or has an unrecognized scope.
+ * @returns {"" | "FRAMEWORK" | "EXAMPLE" | "EXPERIMENTAL" | "INTERNAL" | "PRIVATE" | "TOOLS" | undefined}
+ * A scope kind based on the package's scope (namespace).
+ * Will be an empty string if the package has no scope.
+ * Will be `undefined` if the package has an unrecognized scope.
  */
 const getScopeKindFromPackage = (packageName) => {
 	const packageScope = PackageName.getScope(packageName);
-	if (packageScope === `@fluidframework`) {
+	if (packageScope === "") {
+		return "";
+	} else if (packageScope === `@fluidframework`) {
```

Comment:
Why are these string literals written as template literals?

Reply:
Good question. No idea. Certainly no reason for them to be so now. Will update.